extensions/wikia/AdEngine/js/template/playwire.js

```diff
@@ -1,6 +1,6 @@
 /*global define*/
 define('ext.wikia.adEngine.template.playwire', [
-	'ext.wikia.adEngine.video.playwire',
+	'ext.wikia.adEngine.video.player.playwire.playwire',
```

Comment:
in porvata we have `ext.wikia.adEngine.video.player.porvata` even though we have the same directory structure as with playwire. @nandy-andy what do you think about small exception and removing the second playwire?

First playwire stands for the directory name and the second playwire stands for file name.